### PR TITLE
Updated 2021 Colors for Java

### PR DESCRIPTION
**Pull request by napsterv**
# Description
-Updated text, Ctrl Clickable, Keyword colors
-Method parameter, generic type parameter, Class/Interface colors for Java

⚠️ Remember to update changelog!

## Type of change

- 🐛 bug fix

## Mentions

- [@mention](https://help.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax#mentioning-people-and-teams)

### CODE REVIEW

**Comment by bhushan** [comment]
@napsterv asking few more people to review

**Comment by napsterv** [comment]
> @napsterv asking few more people to review

Alrighty @bhushan

File: src/main/resources/GitHub_Light.xml [hunk 1]
```diff
@@ -1,1090 +1,1099 @@
 <scheme name="GitHub Light" version="142" parent_scheme="Default">
-    <metaInfo>
```
Comment (bhushan) [inline src/main/resources/GitHub_Light.xml:2]: @napsterv update exact changes which you want to be changed.. don't include formatting changes.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 # GitHub Theme Changelog
 
 ## [Unreleased]
+Fix Java colors for 2021
```
Comment (bhushan) [inline CHANGELOG.md:6]: also update https://github.com/jiyuan0125/github-theme-for-jetbrains/blob/16bf1de6b2c5806a953516ce83121f5116132d14/gradle.properties#L6 this to 1.1.4

**Review by bhushan — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (bhushan) [reply CHANGELOG.md:6]: also please update exact changes under unreleased section as fix should not be for 2021 only

File: src/main/resources/GitHub_Light.xml [hunk 3]
```diff
@@ -1,1090 +1,1099 @@
...
+    <option name="com.plan9.REGISTER">
+      <value>
+        <option name="FOREGROUND" value="e36209" />
+      </value>
+    </option>
+    <option name="org.toml.KEY">
+      <value />
+    </option>
+  </attributes>
+</scheme>
```
Comment (bhushan) [inline src/main/resources/GitHub_Light.xml:1099]: should be new line at the end of file

**Review by bhushan — CHANGES_REQUESTED** [review]

**Comment by napsterv** [comment]
Will send a separate PR!

**Comment by bhushan** [comment]
@napsterv

**Comment by napsterv** [comment]
> @napsterv 

Thanks bro!

**Pull request closed without merging**